README.md

```diff
@@ -100,7 +105,7 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+a fork is a copy of a repo placed into another github account.  A clone is a direct copy of a repo into another directory.
 ```
```

Comment:
A clone is a direct copy, but specifically its downloading it a full directory/repo to your local computer